### ISSUE TITLE
Theme personal resource display

### ISSUE THREAD

**Issue by ImogenBits** [ImogenBits/tidyplates#7]
Add support for the personal resource display "namplate" to also be styled. Apparently there is an intentional decision not to do this [here](https://github.com/ImogenBits/tidyplates/blob/17b88f65b718a56db73dfd55c7cf1cdb2f3631e8/TidyPlates/TidyPlatesCore.lua#L962-L967) but I don't really see a reason why we wouldn't support the option to do so.

**Comment by wickermoon** [comment]
Yes, I would love this feature. The UI would be quite simple - for me at least. The buffs should have the same design as the debuffs on enemies. I like the elongated ones, but an option to switch between those and the original - just like with the target debuffs - would be fine as well.

The Healthbar should look like the nameplate for enemies, but with a slimmer border, I would say. Maybe - if possible - with a different color (choosable?) to the target nameplates (green is fine, but e.g. blue would also be nice). The Resource bar could actually be as slim as the original PRD resource bar and - for me at least - doesn't need any text (but would be a nice addition).

The text on the health bar would just be "Health Current / Health Total (X%)", just like the bar on enemy nameplates. I'll try to create a mock-up for it.

**Comment by wickermoon** [comment]
So, I've tested around a bit with the piece of code you provided and I happened to create a PRD, which is already nice, but not exactly what I want, but it's amazing how easy it was. :3 What I found out - and this might be the solution to issue #23 - is that if you deactivate Nameplates for Players, Tidy Plates might also sometimes accidentally create a nameplate for yourself, puts it on top of the vanilla PRD and then removes the visibility, but I'm not 100% sure.

**Comment by wickermoon** [comment]
![image](https://user-images.githubusercontent.com/15907070/107966421-1efeb880-6fac-11eb-99d5-c7404d5f165b.png)

Okay, so this is kinda how I envisioned it, but the border is a bit too thick and the name isn't necessary. Also, the resource bar should be the same size (this is the original resource bar)...and I guess that would be it? If there's a separate bar template for the player's PRD, I can probably theme it myself and send you a picture of how I want it, but at the moment it's a bit difficult, as this bar shares its looks with any other "friendly players" bar.

**Comment by ImogenBits** [comment]
thank you so much for all the details! I'll try to see what I can do to actually make it happen for everyone
